src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -69,6 +69,10 @@ private static void setupDrivers(Application application) {
 
 		driverComposite.add(new LineDriverAdapter(drawerController, LineFactory.getSpecialLine(), "special"));
 		DriverFeature.addDriver("Special line Simulator", driverComposite.getDriver(2));
+
+		DriverFeature.addDriver("Driver composite", driverComposite);
```

Comment:
OK, but revert changes to previous addDriver calls (to make code more clear).

Reply:
Reverted.